yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -163,6 +163,24 @@ func (d *TableIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 						))
 				}
 			}
+
+			if c.ConstraintType == queryparser.PRIMARY_CONSTR_TYPE {
+				//Report PRIMARY KEY (createdat timestamp) as hotspot issue
+				for idx, col := range c.Columns {
+					columnWithHotspotTypes, tableHasHotspotTypes := d.columnsWithHotspotRangeIndexesDatatypes[table.GetObjectName()]
```

Comment:
Note for future: 
I feel like the `columnsWithHotspotRangeIndexesDatatypes` is getting a bit messy. 

There are multiple fields in `ParserIssueDetector` that keep track of the column types for different use-cases: 
- columnsWithUnsupportedIndexDatatypes
- columnsWithHotspotRangeIndexesDatatypes
- compositeTypes
- enumTypes
- jsonbColumns

We should standardise this before it gets even more complicated.
The parserIssueDetector that scans all the tables can just simply track all column types in a single struct, and then everything else can be derived from that. 
(let's discuss this)